app/controllers/chat_channels_controller.rb

```diff
@@ -172,6 +174,41 @@ def channel_info
     render json: { error: "not found", status: 404 }, status: :not_found
   end
 
+  def create_channel
```

Comment:
To make CodeClimate happy — and because this is a pretty long method — we could potentially introduce a `ChatChannels::Create` service.

Reply:
@citizen428  I was thinking about the same approach but still, there will be 21 methods.

Reply:
@citizen428 I fixed the code-climate issues. I found a few methods in the controller that can be moved under the helpers.

Reply:
We are getting into personal preference territory here, but I personally strongly dislike helpers because the became a messy global dumping ground. If this is for building JSON data specifically, it sounds like a job for a serializer.